src/js/controller/AppController.js

```diff
@@ -41,20 +41,30 @@ L.DNC.AppController = L.Class.extend({
         this.dropzone.fileReader.on( 'fileparsed', this._handleParsedFile.bind( this ) );
         this.forms.on( 'submit', this._handleFormSubmit.bind(this) );
         this.menu.geo.on( 'click', this._handleGeoClick.bind(this) );
-        this.ops.geox.on( 'created', this._handleGeoResult.bind(this) );
```

Comment:
`ops.geox` no longer sends back a `'created'` signal, it just [returns it's results directly](https://github.com/cugos/drop-n-chop/pull/79/files#diff-dd6550659eb3f562e3daf4cb53c84280R57).